iptables/actions.go

```diff
@@ -33,6 +37,12 @@ func (g GotoAction) String() string {
 	return "Goto->" + g.Target
 }
 
+func (g GotoAction) ReferredToChain() string {
+	return g.Target
+}
+
+var _ Referrer = (*GotoAction)(nil)
```

Comment:
I see CI is passing, so I'm curious why that's OK.  I'd expect an error like "*GotoAction does not satisfy Referrer interface, but GotoAction does".

Reply:
I believe non-pointer receiver methods can be called on pointers but not the other way around.  That said, I've changed this to use the non-pointer type since that's what I actually require.